apps/datahub/src/app/record/record-metadata/record-metadata.component.html

```diff
@@ -120,9 +120,35 @@
     </div>
 
     <div *ngIf="displayApi$ | async">
-      <datahub-record-apis class="block mt-6"></datahub-record-apis>
+      <datahub-record-apis
+        (selectedLink)="selectCard($event)"
```

Comment:
wondering if it could go in mdview state .. any thought @tkohr @jahow ?

Reply:
I think I'd rather keep it in the `record-metadata` "state" like all the `displaySomething$` observables. But maybe you see some advantage in the facade for later needs?